AppDashboard/templates/layouts/panel.html

```diff
@@ -0,0 +1,26 @@
+<div class="panel panel-default" id="panel{{ panel_count }}">
```

Comment:
I would recommend basing the panel names on their heading name rather than their count. You could include a attribute-friendly ID in the JSON structure.